circuits/circom/prover/index.ts

```diff
@@ -0,0 +1,67 @@
+import 'express-async-errors'
+import { execSync } from 'child_process'
+import { buildPoseidon } from 'circomlibjs'
+import express from 'express'
+import asyncHandler from 'express-async-handler'
```

Comment:
I think [express-async-handler](https://github.com/davidbanham/express-async-errors/blob/master/index.js) and [express-async-errors](https://github.com/davidbanham/express-async-errors/blob/ba7b6e83d2ee832cdf944969672f50be845b3619/index.js#L14-L20) do the same thing no?. So having them both is redundant.
I prefer `express-async-errors` as it is just one import. Instead of wrapping all your handlers.

Anyway it is not a blocker for merging.
Let's clean that later.